functions/submit-finding.js

```diff
@@ -115,13 +121,17 @@ exports.handler = async (event) => {
   };
 
   try {
+    const markdownPath = `data/${handle}-${risk}-${Date.now()}.md`;
```

Comment:
Maybe move this definition under `const path`, and:

```const markdownPath = `data/${handle}-${issueId}.md`;```

Reply:
Good point. I actually planned on this first, but I wanted the issue to be able to link to the markdown file, and if I put the issue id in the markdown filename, then I can't know the path to the file until after the issue is created.

Reply:
Oof. Yeah. Makes sense. Maybe there's a workaround.. because this makes programmatically getting the linked text a bit.. rough.

* Post the link as a comment?
* Alter the issue body with the link once the ID's known?